chains/addresses.yaml

```diff
@@ -45,7 +45,7 @@ alephzeroevmmainnet:
   interchainAccountIsm: "0x4d264424905535E97396Db83bd553D0d73A4EF9d"
   interchainAccountRouter: "0x26A29486480BD74f9B830a9B8dB33cb43C40f496"
   interchainGasPaymaster: "0x9c2214467Daf9e2e1F45b36d08ce0b9C65BFeA88"
-  interchainSecurityModule: "0x75177B4869d1DD30A1717db94d999cBe5C228211"
+  interchainSecurityModule: "0xAF6B4222dB762a21E5DB86e5a096A9f61ff8536B"
```

Comment:
why all these changes?

Reply:
we locked down some repo settings so the GH action that does these updates on main automatically is not fully working - the `yarn build` updates the concatenated addresses/metadata file and that's not run since merging the 2 recent ISM update PRs

Reply:
no longer necessary now that CI is fixed - removing from PR to cleanup diff